SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+All versions of Openshot are security supported. This means we keep all versions of Openshot update to date with the latest security features around.
+
+
+
+## Reporting a Vulnerability
+
+Use this section to tell people how to report a vulnerability.
+If you find a security vulnerability we would like to here from you asap.
```

Comment:
```suggestion
If you find a security vulnerability we would like to hear from you as soon as possible.
```

(It's a bad idea to assume that colloquialisms like "asap" are universally understood.)